src/main/javaServlets/ReviewServlet.java

```diff
@@ -0,0 +1,63 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.*;
+import java.io.IOException;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns reviews.*/
+@WebServlet("/review")
+public class ReviewServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Consider all data involved as preliminary
+    Query query = new Query("Review").addSort("date", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<String> reviews = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      // Get desired information from Datastore
+      long id = entity.getKey().getId();
+      String message = (String) entity.getProperty("message");
+      Date timestamp = (Date) entity.getProperty("date");
+      reviews.add(message + " - " + timestamp.toString());
```

Comment:
I think we're gonna end up retrieving more properties, but this is an easy skeleton for adding that in, nicely done